### PR TITLE
doc: release: devicetree for v3.3

### PR DESCRIPTION
**Pull request by mbolivar-nordic**
The bindings content was mostly generated with a script, but there were some manual changes and additions done to the output in some cases.

The API content was generated by hand.

### CODE REVIEW

**Review by galak — DISMISSED** [review]

**Comment by galak** [comment]
Did you auto-gen this or do it by hand?

**Comment by mbolivar-nordic** [comment]
@galak can you take another look?

**Review by dkalowsk — APPROVED** [review]
Nice auto-generation

File: doc/releases/release-notes-3.3.rst [hunk 1]
```diff
@@ -1064,30 +1064,827 @@ USB
...
 
-* Bindings
+API
+===
 
-  * New:
+New general-purpose macros:
 
-    * :dtcompatible:`zephyr,flash-disk`
+- :c:macro:`DT_FOREACH_PROP_ELEM_SEP_VARGS`
```
Comment (stephanosio) [inline doc/releases/release-notes-3.3.rst:1072]: We have been recommending using `*` for bullet points for the sake of consistency, but I can batch-fix these later.

**Review by stephanosio — APPROVED** [review]
Looks great, thanks.

File: doc/releases/release-notes-3.3.rst (hunk 1 above)
Comment (mbolivar-nordic) [reply doc/releases/release-notes-3.3.rst:1072]: > We have been recommending using `*` for bullet points

FYI, I have updated the script accordingly; thanks for the heads-up

**Pull request merged**